miner/worker.go

```diff
@@ -1010,11 +1013,3 @@ func (w *worker) isIstanbulEngine() bool {
 	_, isIstanbul := w.engine.(consensus.Istanbul)
 	return isIstanbul
 }
-
```

Comment:
How come this function is removed?

Reply:
It was added on 1.9.9 for testing purposes... but then when adapting the tests to use our fake consensus engine it was not needed anymore, so the linter complained about not being used